common/services/playlistResource.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Created by a.ovsiannikov on 7/9/2015.
+ */
+(function(){
+    "use strict";
+
+    angular
+        .module("common.services")
+        .factory("playlistResource", ["$resource",playlistResource]);
+
+    function playlistResource($resource) {
+        return $resource(
+                "https://content.googleapis.com/youtube/v3/playlists",null
+                ,
+                {
+                    'get':    {method:'GET',params:{id:'@id'}},
+                    'save':   {method:'POST'},
+                    'query':  {method:'GET', params:{part: 'contentDetails', mine: 'true'}, withCredentials:true},
```

Comment:
'part' parameter value should be 'snippet', and also there should be parameter 'maxResults' = 50, because by default google returns only 5 results.